### DISCUSSION TITLE
Add option to align to the right in table component

### DISCUSSION THREAD

**Discussion by mrmokwa** [vmware/clarity#5776] [Ideas] [answered]
Currently, the table has the default value of align itens in the center and we can change it to align to the left, as we can see in the image below.

![image](https://user-images.githubusercontent.com/12090931/112483666-04142680-8d58-11eb-9d3b-ab1e46d42526.png)

It would be nice to have an option to align to the right. I would use it to display numeric values.

![image](https://user-images.githubusercontent.com/12090931/112483983-4a698580-8d58-11eb-83ea-fcdfad3e908f.png)

**Comment by mathisscott** [comment]
@gnomeontherun : Is this something that will be possible with the table work you are doing?

**Comment by gnomeontherun** [comment, marked as answer]
This is already in the Core table I'm working on as an option, but the workaround is to create your own CSS rules that align text as needed.

**Reply by mrmokwa** [reply]
Just a curious question... is the Core like a base structure for the Angular Components?
Does it mean that it will be available for "Angular Components - Table"?

**Reply by mathisscott** [reply]
@mrmokwa 
Clarity Core is a web components-based (framework agnostic) implementation of Clarity. It's different from Clarity Angular/Clarity UI. Here are some links if you are interested in more information...

https://medium.com/claritydesignsystem/level-up-your-application-by-adopting-clarity-core-8a5f3f863139

https://medium.com/claritydesignsystem/design-system-performance-with-clarity-core-web-components-fbab56516f30

https://medium.com/claritydesignsystem/clarity-5-0-jump-start-with-core-web-components-dcb22a51222e

https://clarity.design/core-components/

https://clarity.design/storybook/core/

So this will not be available for the `Angular Components - Table` but would be available for Tables under Core. To get a sense of what that would look like, I recommend checking out the Core ul/ol lists. Tables in core will be likewise CSS driven. 

https://clarity.design/core-components/list/code/
